tests/Microsoft.Identity.Web.AotCompatibiliy.TestApp/Microsoft.Identity.Web.AotCompatibiliy.TestApp.csproj

```
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <TargetFrameworks>net8.0;</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+    <PublishAot>true</PublishAot>
+    <TrimmerSingleWarn>false</TrimmerSingleWarn>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- List of the assemblies to analyze.
+
+      FullAnalyze indicates whether to analyze the whole assembly (i.e. TrimmerRootAssembly).
+      Other referenced assemblies will only have code referenced from Program.cs analyzed.
+      
+    -->
+    <CompatibleReference Include="Microsoft.Identity.Web" FullAnalyze="true"/>
+    <CompatibleReference Include="Microsoft.Identity.Web.Azure" FullAnalyze="true" />
+    <CompatibleReference Update="@(CompatibleReference)" Path="..\..\src\%(Identity)\%(Identity).csproj" />
+
+  </ItemGroup>
+  
+  <ItemGroup>
+    <ProjectReference Include="@(CompatibleReference->'%(Path)')" />
+
+    <TrimmerRootAssembly Include="@(CompatibleReference-&gt;WithMetadataValue('FullAnalyze','true'))" />
+  </ItemGroup>
```

I don't think we need this "FullAnalyze" ability anymore. It was used in https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/ last year because we only wanted to analyze part of `Microsoft.IdentityModel.JsonWebTokens` at the time. (and we didn't have the script which allows us to baseline the existing warnings).

Going forward for new repos, I think we can use the following pattern:

```suggestion
  <ItemGroup>
    <TrimmerRootAssembly Include="Microsoft.Identity.Web" />
    <TrimmerRootAssembly Include="Microsoft.Identity.Web.Azure" />

    <TrimmerRootAssembly Update="@(TrimmerRootAssembly)" Path="..\..\src\%(Identity)\%(Identity).csproj" />
    <ProjectReference Include="@(TrimmerRootAssembly->'%(Path)')" />
  </ItemGroup>
```